controller/src/main/java/io/pravega/controller/server/security/auth/handler/AuthHandlerManager.java

```diff
@@ -157,7 +157,7 @@ public void registerInterceptors(ServerBuilder<?> builder) {
                 ServiceLoader<AuthHandler> loader = ServiceLoader.load(AuthHandler.class);
                 for (AuthHandler handler : loader) {
                     try {
-                        handler.initialize(serverConfig);
+                        handler.initialize(serverConfig.toAuthHandlerProperties());
```

Comment:
as discussed, we should keep the interface for auth handler unchanged so that dependents do not have to update. 
we can add a new method in the serverconfig to get properties.

Reply:
Sure, I've now modified the implementation, as suggested, to avoid the change in the interface here - handler.initialize() continues to take `ServerConfig` as input.